### PR TITLE
Disable report size trimming in ScoreCard.

### PR DESCRIPTION
**Pull request by isoos**

### CODE REVIEW

File: app/lib/scorecard/backend.dart [hunk 1]
```diff
@@ -218,7 +218,9 @@ class ScoreCardBackend {
         if (size > _reportSizeDropThreshold) {
           _logger.reportError(
               '$reportType report exceeded size threshold ($size > $_reportSizeWarnThreshold)');
-          return true;
+          // Size trimming is temporarily disabled.
+          // TODO: re-enable after fixing the underlying issues with the verbose reports and package:appengine.
+          return false;
```
Comment (sigurdm) [inline app/lib/scorecard/backend.dart:223]: How about we instead make the truncated report something meaningful?
```
gzip.encode(json.encode(Report(sections: [])));
```

Comment (jonasfj) [reply app/lib/scorecard/backend.dart:223]: Further down we actually have:
```
      if (sizeCheck(ReportType.pana, scoreCard.panaReportJsonGz)) {
        // TODO: replace with something meaningful
        scoreCard.panaReportJsonGz = <int>[];
      }
```

What you are proposing seems meaningful to me :D


Or we just make a single artificial section that says: "Failed to run pana" or something like that.

**Review by jonasfj — COMMENTED** [review]
Don't we need to do this? Or is enough that we have trimmed output from various commands?

File: app/lib/scorecard/backend.dart (hunk 1 above)
Comment (isoos) [reply app/lib/scorecard/backend.dart:223]: I planned to have something like that (as the TODO suggests it), but there may be a side effect which I don't recall now, so I'd rather just turn it off for this release, and revisit what side effects we may have.

**Review by jonasfj — APPROVED** [review]

**Pull request merged**